shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.h

```diff
@@ -367,61 +395,83 @@ class FlutterPlatformViewsController {
   fml::scoped_nsobject<UIViewController<FlutterViewResponder>> flutter_view_controller_;
   fml::scoped_nsobject<FlutterClippingMaskViewPool> mask_view_pool_;
   std::map<std::string, fml::scoped_nsobject<NSObject<FlutterPlatformViewFactory>>> factories_;
-  std::map<int64_t, fml::scoped_nsobject<NSObject<FlutterPlatformView>>> views_;
-  std::map<int64_t, fml::scoped_nsobject<FlutterTouchInterceptingView>> touch_interceptors_;
-  // Mapping a platform view ID to the top most parent view (root_view) of a platform view. In
-  // |SubmitFrame|, root_views_ are added to flutter_view_ as child views.
-  //
-  // The platform view with the view ID is a child of the root view; If the platform view is not
-  // clipped, and no clipping view is added, the root view will be the intercepting view.
-  std::map<int64_t, fml::scoped_nsobject<UIView>> root_views_;
-  // Mapping a platform view ID to its latest composition params.
-  std::map<int64_t, EmbeddedViewParams> current_composition_params_;
-  // Mapping a platform view ID to the count of the clipping operations that were applied to the
-  // platform view last time it was composited.
-  std::map<int64_t, int64_t> clip_count_;
```

Comment:
Thank you for your service, `clip_count_` 🫡. At some point, I assume you must have done something to deserve being here.

Reply:
🫡